### PR TITLE
Add EE E2E Test Coverage Support and Fix Edit Stage Instance Bug

### LINKED ISSUE #2287: Cannot change Instance on a Pipeline Stage
### Current Behavior

> found in call with @biancode

When trying to edit a Stage, and updating the attached Instance, the changes are not persisting after "save"

### Expected Behavior

Editing the instance attached to a Stage within a Pipeline should persist after "save".

### Steps To Reproduce

- Create 2 x Instances
- Create a Pipeline
- Create a Stage & assign an Instance
- Edit the stage, and assign the other instance.
- Click "save"

Note that the Instance has not changed.

### Environment

- FlowForge version: 1.8.0

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by Pezmc**
## Description

This PR has two halves:

 - Fixing the bug when changing a stages instance
 - Adding support for EE E2E test coverage to flowforge

Cypress commands are now as follows:

```
  cy:web-server
    npm-run-all --parallel cy:web-server:os cy:web-server:ee
  cy:web-server:os
    node ./test/e2e/frontend/test_environment_os
  cy:web-server:ee
    node ./test/e2e/frontend/test_environment_ee
  cy:run
    npm-run-all --parallel cy:run:os cy:run:ee
  cy:run:os
    cypress run --config-file ./config/cypress-os.config.js
  cy:run:ee
    cypress run --config-file ./config/cypress-ee.config.js
  cy:open:os
    cypress open --config-file ./config/cypress-os.config.js
  cy:open:ee
    cypress open --config-file ./config/cypress-ee.config.js
```

A developer would run `cy:web-server(:os/:ee)` and then `cy:open:os` or `cy:open:ee` to work on the tests.

To run the tests `cy:web-server(:os/:ee)` and then `cy:run(:ee/:os)`

The EE E2E tests live in `test/e2e/frontend/cypress/tests-ee`

## Related Issue(s)

Fixes #2287 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by Pezmc** [comment]
This PR now has test coverage. Importantly this is EE test coverage.

Going forward, EE E2E test coverage lives in: `test/e2e/frontend/cypress/tests-ee`

You can run EE E2E test coverage using: `npm run cy:web-server:ee` and `npm run cy:run:ee`

OS is also now scoped to: `npm run cy:web-server:os` and `npm run cy:run:os`

`cy:run`, `cy:web-server` will run both EE and OS in parallel.

**Comment by joepavitt** [comment]
worth updating PR title just to reflect the scope of work done here now?

File: frontend/src/pages/application/Pipeline/create.vue [hunk 1]
```diff
@@ -1,26 +1,12 @@
 <template>
-    <Teleport
```
Comment (joepavitt) [inline frontend/src/pages/application/Pipeline/create.vue:2]: Think I'm going mad, but can't work out (even though I can see it works) how we got away with deleting all of these Teleports?

Comment (joepavitt) [reply frontend/src/pages/application/Pipeline/create.vue:2]: discussed ona c all, was code not doing anything, not sure how it got here in the first place.

**Review by joepavitt — APPROVED** [review]

**Pull request merged**